### PR TITLE
feat(dedicated.netapp): change some texts

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/efs-vrack-services`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13895
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by dectotam — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/netapp/src/dashboard/vrack-configuration/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   "netapp_vrack_configuration_vrack_field_placeholder": "Sélectionnez un vRack",
   "netapp_vrack_configuration_no_vrack_field": "Vous n'avez pas de vRack disponible",
   "netapp_vrack_configuration_create_vrack_text": "Souhaitez-vous créer un nouveau vRack ?",
   "netapp_vrack_configuration_create_button": "Créer",
   "netapp_vrack_configuration_associate_button": "Associer",
   "netapp_vrack_configuration_cancel_button": "Annuler",
   "netapp_vrack_configuration_success_message": "Vos paramètres réseaux sont en cours de création",
-  "netapp_vrack_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à votre vRack services : <br> {{message}}<br> request_id: {{requestId}}"
+  "netapp_vrack_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à vRack services : <br> {{message}}<br> request_id: {{requestId}}"
```
Comment (anooparveti) [inline packages/manager/modules/netapp/src/dashboard/vrack-configuration/translations/Messages_fr_FR.json:12]: **nitpick:** `<br> {{message}}<br> request_id: {{requestId}}` can be moved out of trans.
```suggestion
  "netapp_vrack_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à vRack services : "
```

File: packages/manager/modules/netapp/src/dashboard/network-configuration/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-  "netapp_network_configuration_no_subnet_field": "Vous n'avez pas de sous-réseau associé à ce vRack Services",
+  "netapp_network_configuration_no_subnet_field": "Vous n'avez pas de sous-réseau associé à vRack Services",
   "netapp_network_configuration_no_subnet_text": "Souhaitez-vous créer un nouveau sous-réseau ?",
-  "netapp_network_configuration_create_button": "Créer",
+  "netapp_network_configuration_create_button": "Activer",
   "netapp_network_configuration_configure_button": "Configurer",
   "netapp_network_configuration_cancel_button": "Annuler",
   "netapp_network_configuration_success_message": "Vos paramètres réseaux sont en cours de création",
-  "netapp_network_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à votre vRack services : <br> {{message}}<br> request_id: {{requestId}}"
+  "netapp_network_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à vRack services : <br> {{message}}<br> request_id: {{requestId}}"
```
Comment (anooparveti) [inline packages/manager/modules/netapp/src/dashboard/network-configuration/translations/Messages_fr_FR.json:21]: **nitpick:** `<br> {{message}}<br> request_id: {{requestId}}` can be moved out of trans
```suggestion
  "netapp_network_configuration_vrack_association_warning": "Un problème est survenu lors de l'association de votre vRack à vRack services :"
```

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js [hunk 3]
```diff
@@ -143,13 +143,11 @@ export default class OvhManagerNetAppNetworkConfigurationCtrl {
...
               'alert-warning',
-              this.$translate.instant(
+              `${this.$translate.instant(
                 'netapp_network_configuration_vrack_association_warning',
-                {
-                  message: data.vrackAssociationStatus.message,
-                  requestId: data.vrackAssociationStatus.requestId,
-                },
-              ),
+              )}<br> ${data.vrackAssociationStatus.message}<br> request_id: ${
```
Comment (anooparveti) [inline packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js:148]:
```suggestion
              )}<br /> ${data.vrackAssociationStatus.message}<br> request_id: ${
```

File: packages/manager/modules/netapp/src/dashboard/vrack-configuration/controller.js [hunk 4]
```diff
@@ -66,13 +66,12 @@ export default class OvhManagerNetappVrackConfigurationCtrl {
...
               'netapp_vrack_configuration_vrack_association_warning',
-              {
-                message: error?.data?.message || error.message,
-                requestId: error.headers('X-Ovh-Queryid'),
-              },
-            ),
+            )} <br> ${error?.data?.message ||
+              error.message} <br> request_id: ${error.headers(
+              'X-Ovh-Queryid',
+            )}`,
```
Comment (anooparveti) [inline packages/manager/modules/netapp/src/dashboard/vrack-configuration/controller.js:74]:
```suggestion
            )} <br /> ${error?.data?.message ||
              error.message} <br /> request_id: ${error.headers(
              'X-Ovh-Queryid',
            )}`,
```

File: packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js (hunk 3 above)
Comment (qpavy) [reply packages/manager/modules/netapp/src/dashboard/network-configuration/controller.js:148]: There is no need to close sinlge tags in html 5 : https://www.w3schools.com/tags/tag_br.asp

File: packages/manager/modules/netapp/src/dashboard/vrack-configuration/controller.js (hunk 4 above)
Comment (qpavy) [reply packages/manager/modules/netapp/src/dashboard/vrack-configuration/controller.js:74]: same as https://github.com/ovh/manager/pull/11112#discussion_r1515768836

**Review by anooparveti — APPROVED** [review]

**Pull request merged**